scarb/src/compiler/compilers/lib.rs

```diff
@@ -74,6 +76,18 @@ impl Compiler for LibCompiler {
             })?;
         }
 
+        if props.sierra_text {
+            let mut file = target_dir.open_rw(
+                format!("{}.sierra", unit.target().name),
+                "output file",
+                ws.config(),
+            )?;
+            file.write_all(sierra_program.to_string().as_bytes())?;
+            ws.config()
+                .ui()
+                .warn("textual Sierra lib target is being discouraged, use JSON Sierra instead")
```

Comment:
I don't think this warning is desired. It's not bad to use this output kind if you know what you do. And if users did pay attention to the docs in order to find this feature, then I conclude they do know what they do? Wdyt? I suggest removing this.

Reply:
Your worries are definitely correct! But I think we can tackle this from the opposite side: by pushing tools to migrate to JSON :)